docs/faq.rst

```diff
@@ -0,0 +1,37 @@
+Frequently Asked Questions
+===============================
+
+.. glossary::
+
+	Is the information or data I add open access?
+		No, nothing is open access on CERN Analysis Preservation (except for the projects own source code). It is designed to be a save environment for CERN physicists to use from the very beginning of starting their analysis and at any given moment in its lifetime. It is also meant to increase this lifetime. Access to your analysis is always restricted to your collaboration and the permissions you assign.
+		
+		If you are searching for a service providing open access data or you have some data to share, you may want to check out `http://opendata.cern.ch/ <CERN Open Data>`_.
+
+	With all this rerunning, cloning and reusing - where is my work? Are you trying to automate analyses?
+		No, we do not want nor can we do that. An analysis is an extremely complex research work which is precisely the reason why we care to preserve it. After all, if it were easy to automate the process of an analysis, why store it? What we are trying to do is to help you with repetitive tasks, help you find information you need and support your review and approval process so you can focus on the actual research.
+
+	I keep on hearing this word ... and I do not know what it means. Can you help me?
+		Sure, if you cannot find an explanation :doc:`here <reference-guide/jargon>` simply ask us :doc:`directly <support>` and we will help you.
+
+	
+
+.. - I am an experimentalist, how can I use it?
+
+.. - I am a theorist, how can I use it?
+
+.. - What information do I need to prepare from my analysis? When should I do it in my analysis (whenever there is an update that you need to save; we version)
+
+.. - I am continuing my colleagues analysis. How can I reuse the info [clone the record, start editing]
+
+.. - How to use our API with a list of our resources (links/urls they can do requests), descriptions, etc: something like https://developers.facebook.com/docs/graph-api/using-graph-api and also https://about.gitlab.com/handbook/marketing/blog/
+
+.. - As a database provider within LHC collab, how can I contribute to or profit from CAP
+
+.. - Why do I have to enter information into internal databases AND here? - you dont have to; explain
+
+.. - Do I have to use this rather long form or what other options do I have? - shell, API
+
+.. - I can edit, can my collaborators edit my analysis too? - All your collaboration colleagues can read it, only those you invite specifically (personally or e-group) can edit as well
```

Comment:
When talking about editing, it might help in that case to add that in **draft** mode, only the creator can see it (as opposed to "all your collaboration colleagues can read it")